src/App/Command/EnableCommand.php

```diff
@@ -48,11 +48,16 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $enablePackageIds = array_unique(explode(',', $commaSeparatedPackageIds));
         }
 
+        $alreadyEnabledPackages = [];
         $enabledPackages = [];
         foreach ($enablePackageIds as $packageId) {
             if (isset($packages[$packageId])) {
-                $packages[$packageId] = true;
-                $enabledPackages[] = $packageId;
+                if ($packages[$packageId]) {
```

Comment:
Similar to https://github.com/yiisoft/yii-dev-tool/pull/219/files#r947529868.

Reply:
Done.